src/Apps/Artist/ArtistApp.tsx

```diff
@@ -45,21 +59,57 @@ export const ArtistApp: React.FC<ArtistAppProps> = props => {
     })
   }, [])
 
+  const route = findCurrentRoute(props.match)
+
   return (
     <AppContainer>
       <ArtistMeta artist={artist} />
-      <Row>
-        <Col>
-          <ArtistHeader artist={artist} />
-        </Col>
-      </Row>
+      {route.displayNavigationTabs && (
```

Comment:
We store some metadata (`displayNavigationTabs`) on the route so we can use it here to decide whether or not to display the "Back to artist" link or the full set of tabs.